packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -370,11 +370,11 @@ Future<T> showModalBottomSheet<T>({
 ///    sheet.
 ///  * [Scaffold.of], for information about how to obtain the [BuildContext].
 ///  * <https://material.google.com/components/bottom-sheets.html#bottom-sheets-persistent-bottom-sheets>
-PersistentBottomSheetController<T> showBottomSheet<T>({
+PersistentBottomSheetController<void> showBottomSheet({
```

Comment:
ditto

Reply:
i think with this one it's true that there's no current way to provide a return value, but that's probably just an oversight in the api. The close method ideally would take an optional T.